Package.swift

```diff
@@ -21,6 +21,9 @@ let package = Package(
     .executable(name: "Runner", targets: ["Runner"]),
     .executable(name: "Server", targets: ["Server"]),
     .library(name: "Database", targets: ["Database"]),
+    .library(name: "DecodableRequest", targets: ["DecodableRequest"]),
+    .library(name: "EmailAddress", targets: ["EmailAddress"]),
+    .library(name: "FoundationPrelude", targets: ["FoundationPrelude"]),
```

Comment:
I'm adding these libraries because it's what seems to give us targets that we can build in isolation in Xcode...

Reply:
I only see this:

<img width="211" alt="image" src="https://user-images.githubusercontent.com/135203/74678708-9bc83e00-5170-11ea-95b3-9650995b69bb.png">

maybe it's fixed in a beta, but not in 11.3